src/rebar_relx.erl

```diff
@@ -72,13 +71,42 @@ do(Provider, State) ->
 
     {ok, State}.
 
+read_relx_config(State, Options) ->
+    ConfigFile = proplists:get_value(config, Options, []),
+    case ConfigFile of
+        "" ->
+            case {rebar_state:get(State, relx, []), file:consult("relx.config")} of
+                {[], {ok, Config}} ->
+                    ?DEBUG("Using relx.config", []),
+                    Config;
+                {Config, {error, enoent}} ->
+                    ?DEBUG("Using relx config from rebar.config", []),
+                    Config;
+                {_, {error, Reason}} ->
+                    erlang:error(?PRV_ERROR({config_file, "relx.config", Reason}));
+                {RebarConfig, {ok, _RelxConfig}} ->
+                    ?WARN("Found conflicting relx configs, using rebar.config", []),
+                    RebarConfig
+            end;
+        ConfigFile ->
+            case file:consult(ConfigFile) of
+                {ok, Config} ->
+                    ?DEBUG("Using relx config file: ~p", [ConfigFile]),
```

Comment:
same formatting here. Might want to use `~ts` for the output of the file though, so we show the string as-is and in unicode-aware text

Reply:
Fixed in https://github.com/erlang/rebar3/pull/2435/commits/96d61cf5f80a8917b66c73eb41a8ec4fd1773f2e